test/parallel_api/ranges/std_ranges_zip_view.pass.cpp

```diff
@@ -99,6 +99,45 @@ main()
         EXPECT_TRUE(std::get<0>(zip_view_sort[i]) == max_n - 1 - i && std::get<1>(zip_view_sort[i]) == max_n - 1 - i,
             "Wrong effect for oneapi::dpl::ranges::sort with zip_view.");
     }
+#if TEST_DPCPP_BACKEND_PRESENT
+    {
+    const char* err_msg = "Wrong effect for oneapi::dpl::ranges::sort with zip_view and a device policy."
+
+    const int n = big_size;
+    std::vector<int> vals(n), keys(n);
+
+    //test with random number and projection usage
+    std::default_random_engine gen{std::random_device{}()};
+    std::uniform_real_distribution<float> dist(0.0, 100.0);
```

Comment:
The type of keys and values is `int`. Let's align the types of the generator and the data.

Reply:
1) A am not sure that `uniform_real_distribution `work with int. Must be `RealType` according to the standard.
2) Moreover, we already use the same code pattens in another tests, std_ranges_sycl, for example.

Reply:
Note, that we could use `uniform_int_distribution`, instead. 
Anyway, it does not matter much - the value can be converted.